src/pages/User/utils/getUserById.ts

```diff
@@ -0,0 +1,30 @@
+import { User } from 'src/utils';
+import { useRoute, useRouter } from 'vue-router';
+import { getSpecificUser } from 'src/actions';
+import { ref } from 'vue';
+import { Notify } from 'quasar';
+
+export const getUserbyId = async (
+  id = 'b5ca33ca-fd87-4858-a285-e15429c6dd9b'
+) => {
+  const user = ref<User>();
+  const route = useRoute();
+  await getSpecificUser(id).then((res) => {
+    user.value = res;
+
+    if (!user.value) {
+      const router = useRouter();
+
+      Notify.create({
+        message: 'Tuvimos problemas para encontrar al Usuario',
+        type: 'negative',
+      });
+
+      router.back();
+    }
```

Comment:
I think that this error is not reproducible since you've added a guard in the action in case that user.value was empty

https://github.com/Azordev/did-app/pull/71/files#diff-f05dd15e4782b0da3b14cf9406e73fccb97cb4f128c18f0cc414e140d267e282

What about "catch" the error instead, and redirecting in the catch?